src/components/StudentFeedback.vue

```diff
@@ -23,9 +23,38 @@
         </p>
       </div>
     </div>
+    <div v-else>
+      <h1 class="s-h1">There is nothing for this module yet. Stay tuned.</h1>
+    </div>
 
     <section class="notes-container">
-      <h2>Notes section goes here</h2>
+      <p class="feedback-container__category--label">
+        <span class="s-h2">Student Notes</span>
+      </p>
+      <div class="current-notes">
+        <ul>
+          <li v-for="(note, index) in studentNotes" :key="index">{{ note }}</li>
```

Comment:
I'd like to pick your brain over this directive and use of `:key` when we meet up later!

Reply:
in this situation, key is not directive, v-bind is the directive. So it's saying, bind this key.  Just like with React, where it needs a key for unique things we create with mapping. We can chat later about it.